test/e2e/example_k8petstore.go

```diff
@@ -118,30 +118,31 @@ T:
 	for {
 		select {
 		case <-transactionsCompleteTimeout:
-			Logf("Timeout %v reached, transactions not complete.  Breaking!", tick)
+			Logf("PetStore : Completion timeout %v reached, transactions not complete.  Breaking!", tick)
 			break T
 		case <-startupTimeout:
```

Comment:
I'd prefer this case be above `transactionsCompleteTimeout`, since it should come before.